tinygrad/tensor.py

```diff
@@ -180,6 +180,11 @@ def arange(start, stop=None, step=1, **kwargs):
     if stop is None: stop, start = start, 0
     return Tensor.full((math.ceil((stop-start)/step),), step, **kwargs).cumsum() + (start - step)
 
+  @staticmethod
+  def linspace(start: Union[int, float], stop: Union[int, float], steps: int, **kwargs):
+    assert steps > 1, f"steps must be greater than 1, [{steps}] was provided"
```

Comment:
Steps can be 1 in torch

Reply:
Right, it didn't make too much sense to me (who uses it for 1 or 0 steps?!) so wanted to slightly adjust it.
Anyway updated the code + added few corner cases to match the torch api. 
@geohot WDYT?